### ISSUE TITLE
Findable Help SC3.2.6 - access to at least one option?

### ISSUE THREAD

**Issue by guyhickling** [w3c/wcag#1231]
The following words in the SC are difficult to understand:

> if one of the following is available, then access to at least one option is included...

What options, and where? Options for what?

The above extract from the SC is confusing because it refers to the same thing using two quite different phrases. Once you have mentioned something once you don't have to name it again, and certainly should not call it something else which implies the second reference is to something different. The second time we can just refer to it using "that".

The extract also mixes singular and plural. It says "one of the following" (i.e. singular) the first time, but "at least one option" (now allowing for more than one, plural) the second time.

It will be much clearer (and shorter) if it is changed to use "that". I'll start with the wording I suggest in #1228, which used the words "help sources" (but another name involving "help" could be used if preferred). Changing the last few words of the above extract to use "that" would then make it:

> if one of the following four help sources is available, then access to that source is included...

**Comment by JAWS-test** [comment]
Regardless of the perfect wording (which I can't say anything about due to my lack of knowledge of English), the current wording differs in content from your current proposal (if I understand it correctly): At the moment only one help option is required to be linked (even if there are all four), your wording would make it mandatory to link all available help options on every page, right?

**Comment by guyhickling** [comment]
Good point. This is a problem with the rather stilted language used in the SC, but I see what you are getting at. I am not trying to change the meaning at all in any of these issues about comprehension. So we could change my wording slightly to meet your objection and make it: 

> if one of the one of the following four help sources is available, then access to at least one source is included...

Does that cover your point?

The primary aim of this change is to to use the same name (and I chose "help source" but there are other possibilites as well) for the item in both references to it in this clause, so that the reader can clearly see that the same item is being referred to both times - which isn't obvious in the current wording.

**Comment by JAWS-test** [comment]
> Does that cover your point?

To the extent that the old wording says the same as the new one. The question is whether this is really the intention: if there are 2 help methods available, why shouldn't both be linked, but only one.

**Comment by guyhickling** [comment]
I agree with you that this does seem a bit strange, and certainly worth raising the question and I was debating whether to do that. I think I agree with you, though I haven't thought deeply about it yet. 

However that really wants to be raised as a separate GitHub issue as I am not querying the intent of the SC here. This issue, and the others I raised at the same time, is just concentrating on the understandability of the SC in the hopes of achieving a more easily understood wording. 

If you want to start a new issue for your query I'll support it. I don't think I'll have the time at the moment as I have issues I want to raise on other SCs, and I also have an audit to do this week. Also I believe they want to have everything in for 2.2 by the end of July ready for a wide review.

**Comment by JAWS-test** [comment]
> If you want to start a new issue for your query ...

https://github.com/w3c/wcag/issues/1235

**Comment by scha14** [comment]
Draft changes 

> https://github.com/w3c/wcag/pull/1357/files and https://github.com/w3c/wcag/pull/1357/files

**Comment by guyhickling** [comment]
Yes, that changes it to: 

> if one or more of the following help mechanisms is available....

That mentions "help mechanisms" which certainly improves it. In fact mechanisms is probably better than sources. And the "or more" makes it more flexible as well. So go with it!

**Comment by littlea11ywidget** [comment]
This helps a lot. Could we also make the word in 'than at least one option' to 'mechanism' to make it clear that it's referring to the mechanisms? I feel it would help clear up any doubt. It would read like this:

    For single page Web applications or any set of Web pages, if one or more of the following help mechanisms is supported, 
      then access to at least one **mechanism** is included in the same relative order on each page:

**Comment by alastc** [comment]
Hi everyone, we discussed this in a meeting and whilst agreeing with the substance of the comment, it's a hard terminology problem.

We tried: adding 'option' to both ends, and tried the terms mechanism, method, feature, means and ways.

We'd like to propose going with this as the first part of the SC text:
> For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included in the same relative order on each page:

Also given the discussion here, does resolve the issue you raised?

**Comment by littlea11ywidget** [comment]
Hi Alistair, 
I believe this helps. I added the word 'appearing' but you can take it or leave it. :) 

For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included, **appearing** in the same relative order on each page:

**Comment by guyhickling** [comment]
@littlea11ywidget  That is a big improvement. Thank you.

@alastc The new wording accepted in the meeting that you show above is a huge improvement on the original, and covers most of the issues I saw in it initally. And @littlea11ywidget's addition solves one remaining issue I had with it. So that makes it:

> For single page Web applications or any set of Web pages, if one or more of the following ways of finding help is provided, then access to at least one way of finding help is included, appearing in the same relative order, on each page:

I am happy with that.

**Comment by scha14** [comment]
Closing this to address in the issue #1242, where all issues are combined. Please reopen if this remains unaddressed. Thank you!

**Comment by guyhickling** [comment]
Yes, transferring this to 1242 is fine, and that has been comitted, so thank you.
